furms-web-ui/src/main/java/io/imunity/furms/ui/views/site/settings/SettingsView.java

```diff
@@ -169,6 +172,12 @@ private void doSaveAction(Binder<SiteSettingsDto> binder) {
 						.build());
 				refreshBinder(binder);
 				showSuccessNotification(getTranslation("view.site-admin.settings.form.button.save.success"));
+			} catch (DuplicatedNameValidationError e) {
+				findFormField(binder, TextField.class, NAME_FIELD_ID)
+						.ifPresent(textField -> {
+							textField.setErrorMessage(getTranslation("view.site-admin.settings.form.name.validation.unique"));
+							textField.setInvalid(true);
+						});
 			} catch (RuntimeException exception) {
 				showErrorNotification(getTranslation("view.site-admin.settings.form.error.unexpected"));
```

Comment:
Exception details are lost and not logged

Reply:
+1